File: dnsApi/password/nodejs/dns_lookup_pass.js
var http = require('http');

var domain = 'example.com';
var password = 'your_whois_api_password';
var username = 'your_whois_api_username';

var url = 'http://www.whoisxmlapi.com/whoisserver/DNSService?type=_all'
        + '&domainName=' + domain + '&username=' + username + '&password=' + password;

http.get(url, function(response) {
    var str = '';
    response.on('data', function(chunk) { str += chunk; });
    response.on('end', function() { console.log(str); });
}).end();

File: dnsApi/password/python/dns_lookup_pass.py
try:
    from urllib.request import urlopen
except ImportError:
    from urllib2 import urlopen

domain = 'example.com'
password = 'your_whois_api_password'
username = 'your_whois_api_username'

url = 'http://www.whoisxmlapi.com/whoisserver/DNSService?type=_all'\
    + '&domainName=' +domain + '&username=' +username + '&password=' +password

print(urlopen(url).read().decode('utf8'))